### PR TITLE
[ADD] (Optional) Custom policies to attach to the ECS task execution role

### PR DESCRIPTION
**Pull request by ovcharenko**
~~Need to update `td` version after https://github.com/cn-terraform/terraform-aws-ecs-fargate-task-definition/pull/12 merging and releasing~~ done

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -59,10 +59,11 @@ module "td" {
...
 
-  permissions_boundary  = var.permissions_boundary
-  placement_constraints = var.placement_constraints_task_definition
-  proxy_configuration   = var.proxy_configuration
-  volumes               = var.volumes
+  permissions_boundary                    = var.permissions_boundary
+  placement_constraints                   = var.placement_constraints_task_definition
+  proxy_configuration                     = var.proxy_configuration
+  ecs_task_execution_role_custom_policies = var.ecs_task_execution_role_custom_policies
+  volumes                                 = var.volumes
```
Comment (jnonino) [inline main.tf:66]: Please update the module version too!

**Review by jnonino — CHANGES_REQUESTED** [review]

File: main.tf (hunk 1 above)
Comment (ovcharenko) [reply main.tf:66]: Done in 63c3928

**Review by jnonino — APPROVED** [review]

**Comment by ovcharenko** [comment]
@jnonino When to expect new release?

**Comment by jnonino** [comment]
Done

**Pull request merged**